defaults/main.yml

```diff
@@ -39,6 +39,8 @@ apache_mods_enabled:
   - rewrite.load
   - ssl.load
 apache_mods_disabled: []
+apache_confs_enabled: []
```

Comment:
Can you also add a note in the README.md file about these two new options after mods?